infra/bisector.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Uses _bisection to determine which commit a bug was introduced and fixed.
```

Comment:
I don't think this comment should say the name of the function it uses. That is an implementation detail that can change. Instead the module docstring should explain what the module does on a high level.